azure/functions/decorators/cosmosdb.py

```diff
@@ -106,3 +107,103 @@ def __init__(self,
         self.database_name = database_name
         self.collection_name = collection_name
         super().__init__(name=name, data_type=data_type)
+
+
+class CosmosDBInput(InputBinding):
```

Comment:
In the comments, could you please add a description of what is the difference between `CosmosDBTriggerV3` vs `CosmosDBTrigger` so that that info is in the code.

Reply:
Added